content/tracing/services/_index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Services list
+kind: Documentation
+further_reading:
+- link: "/tracing/services/service"
+  text: Learn more about services in Datadog
+- link: "/tracing/services/resource"
+  text: "Dive into your resource performances and traces"
+- link: "/tracing/services/trace"
+  text: "Understand how to read a Datadog Trace"
+---
+
+## Overview
+
+After you have [instrumented your application](/tracing/setup), your reporting services appear on [the APM services page](https://app.datadoghq.com/apm/services).  
+Select a service view detailed performance insights, [read our dedicated service documentation to learn more](tracing/services/service)
+
+{{< img src="tracing/services/services_page.png" alt="Services page" responsive="true" popup="true">}}
+
+## Filtering the service list
+
+The services list is a bird's eyed view of all [services](/tracing/services/service) reporting from your infrastructure. It can be filtered depending on:
+
+* [Your environment](/tracing/setup/environment)
+* [Your service type](#services-types)
+* A query
+
+{{< img src="tracing/services/services_filtering.gif" alt="Services filtering" responsive="true" popup="true" style="width:75%;">}}
+
+### Services types
+
+Every Service monitored by your application is associated with a "Type". This type is automatically determined by Datadog based on your Service name and is applied for you. The "Type" specified the name of the application/framework the Datadog Agent is Integrating with.
```

Comment:
2 things:

- yes we attach a single type to a service, but to define the type value we don't look at the service name, but on the span.type.
- reading that, I'd expect to be told how to change a type (or maybe we do it later in the doc?).

Reply:
We don't do it later in the doc, since it depends of the instrumentation and so the language used by the customer, we can give more detail here and can only hope that the user will be able to find how to setup the `span.type` with his instrumentation